### PR TITLE
BD-1600: Content Blocks for Drag & Drop Editor GA

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_content_blocks.md [hunk 1]
```diff
@@ -43,7 +39,15 @@ You can also add as many Drag & Drop Editor Content Blocks as needed to build ou
 
 ![][2]{: style="float:right;max-width:50%;margin-left:15px;margin-top:15px;"}
 
-To use the Content Block, navigate to your email campaign and select **Edit Email Body**. Click the <i class="fas fa-plus"></i> **Personalization** button. 
+There are two ways to add the Content Block to your email. 
+
+### Rows
+
+First, go to the the **Rows** tab the editor and select **Content Blocks**. Locate your Content Block, and drag and drop the Content Block into the email editor.
```
Comment (colleenwoodford) [inline _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_content_blocks.md:46]: Do you think it complicates things to say something like - "Once added into the email editor, you can make changes to the content block that will not affect the parent content block you created in the Templates & Media section." ?

I'm thinking about making the behavior difference between adding via dragging (can edit, can preview, as soon as dragged into the email no longer connected to the template/media parent block) versus Liquid (can't edit or preview, always tied to the parent block in templates/media) as clear as possible

Comment (lydia-xie) [reply _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_content_blocks.md]: Added 2 callouts to clarify!

**Review by colleenwoodford — APPROVED** [review]

File: _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_content_blocks.md [hunk 2]
```diff
@@ -43,12 +39,28 @@ You can also add as many Drag & Drop Editor Content Blocks as needed to build ou
 
 ![][2]{: style="float:right;max-width:50%;margin-left:15px;margin-top:15px;"}
 
-To use the Content Block, navigate to your email campaign and select **Edit Email Body**. Click the <i class="fas fa-plus"></i> **Personalization** button. 
+There are two ways to add the Content Block to your email. 
+
+### Rows
+
+First, go to the the **Rows** tab the editor and select **Content Blocks**. Locate your Content Block, and drag and drop the Content Block into the email editor. \
```
Comment (KellieHawks) [inline _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_content_blocks.md:46]:
```suggestion
First, go to the the **Rows** tab the editor and select **Content Blocks**. Locate your Content Block, and drag and drop the Content Block into the email editor.
```

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Looks great! One quick edit needed

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**